netbox_onboarding/netbox_keeper.py

```diff
@@ -118,7 +135,11 @@ def ensure_device_manufacturer(
         nb_manufacturer_slug = slugify(self.netdev_vendor)
 
         try:
-            self.nb_manufacturer = Manufacturer.objects.get(slug=nb_manufacturer_slug)
+            search_array = [
+                {"slug__iexact": nb_manufacturer_slug},
+                {"slug": nb_manufacturer_slug}
```

Comment:
Is there a case I'm not thinking of where `slug__iexact` would be a non-match but `slug` would be a match? Or is this redundant?

Reply:
I believe it's redundant 
In the past someone reported an issue where we didn't match on the exact slug search but we couldn't create the new object because the slug was not available see #52